index.html

```diff
@@ -1,380 +1,458 @@
-            feature that you need built or a project that need coding. I'd
-            love to help with it
-          </p>
-        </div>
-        <div class="form-container">
-          <form action="https://formspree.io/f/mayvwled" method="POST" class="contact-form">
-            <input
-              type="text"
-              placeholder="Name"
-              class="input-client-info"
-              name="full-name"
-              minlength="2"
-              maxlength="30"
-              required
-            />
-            <input
-              type="email"
-              name="email"
-              placeholder="example@gmail.com"
-              class="input-client-info"
-              required
-            />
-            <textarea
-              name="message"
-              id="message"
-              placeholder="right your message here"
-              class="client-message"
-              minlength="10"
-              maxlength="500"
-              required
+      </section>
+      <!-- Contact Section -->
+      <section id="contact-section">
+        <div class="content contact-container">
+          <div class="contact-title-container">
+            <h2 class="contact-title">Contact me</h2>
+          </div>
+          <div class="contact-description-container">
+            <p class="contact-description">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that need coding. I'd
+              love to help with it
+            </p>
+          </div>
+          <div class="form-container">
+            <form
+              action="https://formspree.io/f/mayvwled"
+              method="POST"
+              class="contact-form"
             >
-            </textarea>
-            <div class="button-container">
-              <button name="submit" id="button" class="contact-button">Get It Touch
-              </button>
-            </div>
-          </form>
+              <input
+                type="text"
+                placeholder="Name"
+                class="input-client-info"
+                name="full-name"
+                minlength="2"
+                maxlength="30"
+                required
+              />
+              <input
+                type="email"
+                name="email"
+                placeholder="example@gmail.com"
+                class="input-client-info"
+                required
+              />
+              <textarea
+                name="message"
+                id="message"
+                placeholder="right your message here"
+                class="client-message"
+                minlength="10"
+                maxlength="500"
+                required
+              >
+              </textarea>
+              <div class="button-container">
+                <button name="submit" id="button" class="contact-button">
+                  Get It Touch
```

Comment:
- [ ] There is a contrast issue with your button "Get in touch" because when in hover state it does not differentiate from the background. There are a few more contrast issues that I would suggest fixing. You could see the [complete report here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2021-12-07-at-22-15-09-401?failing) :smiley: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/145114651-207a463a-ce85-42af-a384-3ea7ec86c71f.png)

Reply:
Hi, thanks for all your reviews. Just a quick question I saw the fail report but from template 1 the other elements have the same color for the hover effect ... why this is changing now ?

Reply:
Normally it is because the text is unnoticed because of the background color. You could try changing the `font-weight` or `font-size`, also make sure that the color in default and hover state is noticeable considering the background color :pushpin: :wink:

![image](https://user-images.githubusercontent.com/81522435/145118622-ed1bbde1-e7eb-4579-a2e4-6aa9b1022927.png)